act-rules-format.md

```diff
@@ -206,6 +213,71 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Mapping Secondary Requirements ####
+An accessibility requirement is mapped as a secondary requirement in a rule when both of the following are true: 
+
+- (a.) When one or more of the outcomes for a test target is `failed`, the accessibility requirement is <dfn>not satisfied</dfn> or <dfn>further testing is needed</dfn> for the test subject, or
+- (b.) When all of the outcomes are `passed` or `inapplicable`, the accessibility requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn> or <dfn>not satisfied</dfn>.
+
+
+**Temporary**: these are the situations discussed that necessitated a secondary requirements category: 
+- requirement is sticter than the rule (contrast AAA): the accessibility requirement is (a.)<dfn>not satisfied</dfn> when one or more outcomes for a test target is `failed`, and is (b.) <dfn>not satisfied</dfn>when all outcomes for the rule are `passed`.
```

Comment:
I think we need to be careful with language `one or more outcomes for a test target` -> `one or more outcomes for a test subject/test case`.  Additionally `and is (b.) <dfn>not satisfied</dfn>` -> 'and may not be <dfn>not satisfied</dfn>`